default-config.mk

```diff
@@ -0,0 +1,24 @@
+# Do not modify this file, if you want to configure your own environment copy
+# this file in config.mk and modify that file, or defining environment
+# variables using the same names found here.
+
+# Enable services to be run in docker.
+#
+# Possible options: mysql, postgres, minio, inbucket, openldap, dejavu,
+# keycloak and elasticsearch
+#
+# Must be space spearated names.
+#
+# Example: mysql postgres elasticsearch
+ENABLED_DOCKER_SERVICES ?= mysql
```

Comment:
Our `sqlstore` unit tests require both `mysql` and `postgres` to run. Should we enable both by default to allow for passing unit tests?

Reply:
I doubt about that, I can re-enabled everything that was enabled before as default to avoid anybodies workflow to be broken.

About unit tests... to be honest, almost nobody is running mysql/postgres tests locally except while they are working on the store, so... that is very focused to a small subset of our developers.

Said that, I still think is interesting to keep the thing working as was working before, and only add the "configurable" part. On the other hand, that means, the new comers are going to suffer the huge unneeded overhead anyway.

Reply:
other option is to reenable everything that was enable before, except for enterprise (that means, remove elasticsearch and openldap by default), because we have more control on who is developing that with elastic and ldap, and we can talk with them directly.